docs/core/events.md

```diff
@@ -1 +1,50 @@
-# Events
\ No newline at end of file
+# Events
+
+## Pre-Requisite Reading
+
+- [Anatomy of an SDK application](../basics/app-anatomy.md)
+
+## Synopsis
+
+`Event`s are objects that contain information about the execution of the application. They are mainly used by service providers like block explorers and wallet to track the execution of various messages and index transactions. 
+
+- [Events](#events)
+- [EventManager](#eventmanager)
+
+## Events
+
+`Event`s are implemented in the Cosmos SDK as an alias of the [ABCI `event` type](https://github.com/tendermint/tendermint/blob/master/abci/types/types.pb.go#L2661-L2667). They contain:
```

Comment:
we shouldnt use master branch here, these will be moving soon, can we use a commit instead?

Reply:
@fadeev can we do the same thing with TM (i.e. variable interpolation via `config.js`)